screens/Scan/ScanLayout.tsx

```diff
@@ -43,8 +47,42 @@ export const ScanLayout: React.FC = () => {
 
   return (
     <React.Fragment>
+      <VerifyIdentityOverlay
+        isVisible={
+          controller.isVerifyingIdentity &&
+          controller.flowType === FlowType.MINI_VIEW_SHARE_WITH_SELFIE
+        }
+        vc={controller.selectedVc}
+        onCancel={controller.CANCEL}
+        onFaceValid={controller.FACE_VALID}
+        onFaceInvalid={controller.FACE_INVALID}
+      />
+
+      <MessageOverlay
```

Comment:
Move the Message Overlay and IdentityOverlay into a different component and use it in scan layout and Send vc screens